### PR TITLE
add Panel on Hugging Face blog post

### PR DESCRIPTION
**Pull request by sophiamyang**

### CODE REVIEW

File: _blog.yml [hunk 1]
```diff
@@ -2376,3 +2376,15 @@
   tags:
     - audio
     - research
+
+- local: panel-on-hugging-face
+  title: "Panel on Hugging Face"
+  author: philippjfr, sophiamyang
```
Comment (osanseviero) [inline _blog.yml:2382]: One can just have a single author at this level

File: _blog.yml [hunk 2]
```diff
@@ -2376,3 +2376,15 @@
...
+  thumbnail: /blog/assets/panel-on-hugging-face/thumbnail.png
+  date: June 22, 2023
+  tags:
```
Comment (osanseviero) [inline _blog.yml:2385]: Please also add `open-source-collab`

File: _blog.yml [hunk 3]
```diff
@@ -2376,3 +2376,15 @@
   tags:
     - audio
     - research
+
+- local: panel-on-hugging-face
+  title: "Panel on Hugging Face"
+  author: philippjfr, sophiamyang
+  thumbnail: /blog/assets/panel-on-hugging-face/thumbnail.png
```
Comment (osanseviero) [inline _blog.yml:2383]: Could you compress this image please? 1.3MB is a bit large

File: panel-on-hugging-face.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+- user: sophiamyang
+  guest: true
+---
+
+# Panel on Hugging Face
+
+<!-- {blog_metadata} -->
+<!-- {authors} -->
+
+We are thrilled to announce the partnership between Panel and Hugging Face! 🎉 We have integrated a Panel template in Hugging Face Space to help you get started building Panel apps and deploy them on Hugging Face effortlessly.
```
Comment (osanseviero) [inline panel-on-hugging-face.md:16]:
```suggestion
We are thrilled to announce the collaboration between Panel and Hugging Face! 🎉 We have integrated a Panel template in Hugging Face Spaces to help you get started building Panel apps and deploy them on Hugging Face effortlessly. 
```

File: panel-on-hugging-face.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+<!-- {blog_metadata} -->
+<!-- {authors} -->
+
+We are thrilled to announce the partnership between Panel and Hugging Face! 🎉 We have integrated a Panel template in Hugging Face Space to help you get started building Panel apps and deploy them on Hugging Face effortlessly. 
+
+<a href="https://huggingface.co/new-space?template=Panel-Org/panel-template"> <img src="https://huggingface.co/datasets/huggingface/badges/raw/main/deploy-to-spaces-lg.svg"/> </a>
+
+
+## About Hugging Face
+
```
Comment (osanseviero) [inline panel-on-hugging-face.md:22]: I'm ok keeping this, but given it's published in our blog, I would assume people know it already

**Review by osanseviero — APPROVED** [review]
Very nice! 🔥

File: _blog.yml (hunk 3 above)
Comment (julien-c) [reply _blog.yml:2383]: you can use https://tinypng.com/ for instance @sophiamyang

File: panel-on-hugging-face.md (hunk 5 above)
Comment (pcuenca) [reply panel-on-hugging-face.md:22]: Yes, I think it could distract a bit from the message, and we don't usually have a Hugging Face intro in other posts.

Comment (sophiamyang) [reply panel-on-hugging-face.md:22]: Will remove the section. Thanks!

File: _blog.yml (hunk 1 above)
Comment (sophiamyang) [reply _blog.yml:2382]: got it thanks!

File: _blog.yml (hunk 3 above)
Comment (sophiamyang) [reply _blog.yml:2383]: will do thanks!

**Pull request merged**